app/views/handlers/content.py

```diff
@@ -13,8 +17,28 @@ def get_context(self):
         return {
             "block": self.rendered_block,
             "metadata": dict(self._questionnaire_store.metadata),
+            "individual_response": self._individual_response_content(),
         }
 
+    def _individual_response_content(self) -> Mapping:
+        if "individual_response" in self._schema.json:
+            for_list = self._schema.json["individual_response"]["for_list"]
+            list_item_id = self._current_location.list_item_id
+
+            url = url_for(
+                "individual_response.request_individual_response",
+                list_item_id=list_item_id,
+            )
+
+            primary_person_id = self._questionnaire_store.list_store[
+                for_list
+            ].primary_person
+
+            if list_item_id != primary_person_id:
+                return {"url": url}
+
+        return {}
```

Comment:
Why do we return a dict when there is only 1 property?. Can we not just return url or None?.

Reply:
Because we like consistent return types and an empty dict still evaluates to true when compared to None?

Reply:
I am referring to it just returning 1 thing which is a URL. It should return a URL string or None? (which is consistent with pretty much every thing else we do). Makes sense to return a dict if we have multiple properties?.

Reply:
Yeah I see what you mean - it's a hangover from implementing part of the hub story as well. I'll switch it to url/None for this story